internal/pkg/cli/storage_init.go

```diff
@@ -74,6 +90,20 @@ partition key but a different sort key. You may specify up to 5 alternate sort k
 	storageInitDDBLSINameHelp   = "You can use the characters [a-zA-Z0-9.-_]"
 )
 
+// RDS Aurora Serverless specific questions and help prompts.
+var (
+	storageInitRDSInitialDBNamePrompt = "What would you like to name your initial database?"
+	storageInitRDSInitialDBNameHelp   = "The name of the initial database in the cluster."
+
+	storageInitRDSDBEnginePrompt = "Which database engine would you like to use?"
+	storageInitRDSDBEngineHelp   = "The database engine used in the cluster."
+)
+
+// DynamoDB specific constants and variables.
+const (
+	ddbKeyString = "key"
+)
+
 const (
 	ddbStringType = "String"
```

Comment:
Why are those consts prefixed with `ddb` but not categorized as "DynamoDB specific constants and variables." 🤔 Maybe we should change the names if they are common consts?

Reply:
Oh by the comment `// DynamoDB specific constants and variables.` I actually referred to all `const` and `var` that are between this comment and the next comment (i.e. `// RDS Aurora Serverless specific constants and variables.`) I added those comments just so that it's (probably) clearer which chunk of codes correspond to which storage.

Didn't move `ddbKeyString` and `ddbStringType` (as well as `ddbIntType`, `ddbBinaryType`) into the same `const` block because those `const`s are used in different places.